changelogs/2.11.4.md

```diff
@@ -0,0 +1,161 @@
+  index could lead to a fiber hang (gh-10233, gh-10267).
+* Fixed a multi-threading race condition that could cause a writer thread to
+  crash while looking up a tuple format (gh-10278).
+* Fixed a bug when recovery could fail with the error "Invalid VYLOG file:
+  Deleted range XXXX has run slices" or "Invalid VYLOG file: Run XXXX committed
+  after deletion" after an index drop (gh-10277).
+* The `force_recovery` mode can now be used to suppress "Invalid VYLOG file"
+  recovery errors (gh-10292).
+* Fixed a bug when any DDL operation aborted unrelated transactions (gh-10375).
+
+### Replication
+
+* Fixed a bug when a replica could timeout on subscribe if the master had to
+  open a big enough xlog file for that (gh-9094).
+* Fixed a bug that the `box.info.replication[...].downstream.lag` value could be
+  misleading, not updating in time, frozen (gh-9748).
+* Fixed a bug when `box.ctl.demote()` with `box.cfg{election_mode = 'off'}`
+  and an owned synchro queue could simply not do anything (gh-6860).
+* Fixed a bug that allowed the old leader in
+  `box.cfg{election_mode = 'candidate'` mode to get re-elected after resigning
+  himself through `box.ctl.demote` (gh-9855).
+* Fixed the inability to add a new replica to the replicaset if the user has
+  manually made space `_cluster` synchronous. Now the synchronous replication
+  quorum is updated after the `_cluster` change is confirmed by a quorum rather
+  than immediately after persisting the configuration change in the WAL
+  (gh-10087).
+* Fixed a bug when an anonymous replica register or a replica name assignment
+  could fail with an error "LSN for ... is used twice" in release and crash in
+  debug (gh-9916).
+* Fixed anonymous replicas not receiving the synchronous transaction queue state
+  during join (gh-9401).
+
+### LuaJIT
+
+Backported patches from the vanilla LuaJIT trunk (gh-9924). The following issues
+were fixed as part of this activity:
+
+* Fixed `BC_VARG` recording.
+* Fixed `ffi.alignof()` for reference types.
+* Fixed `sizeof()` expression in C parser for reference types.
+* Fixed `ffi.metatype()` for typedefs with attributes.
+* Fixed `ffi.metatype()` for non-raw types.
+* Fixed IR chain invariant in DCE.
+* Fixed OOM errors handling during trace stitching.
+* Fixed `IR_HREF` vs. `IR_HREFK` aliasing in non-`nil` store check.
+* Fixed generation of Mach-O object files.
+* Fixed undefined behavior when negating `INT_MIN` integers.
+* Replaced the numeric values of NYI bytecodes that can't be compiled, with
+  their names in the `jit.dump()`.
+Backported patches from the vanilla LuaJIT trunk (gh-10199). The following
+issues were fixed as part of this activity:
+
+* Fixed GC marking of the cdata finalizer table.
+
+### Lua
+
+* Fixed a corner case when the netbox connection's worker fiber could crash
+  (gh-9621).
+* Fixed a bug that caused a `net.box` connection to crash after being closed
+  from the connection's `on_schema_reload` trigger (gh-9621).
```

Comment:
Since there are `URI` and `YAML` subsections, we can wrap these two items to the `net.box` subsection for readability.